qemu/tests/usb_device_check_negative.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from virttest import (env_process,
+                      error_context,
+                      virt_vm)
+from qemu.tests.usb_common import parse_usb_topology
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    The usb devices nagetive test
+
+    1) Boot up guest invalid usb devices
+    2) Verify qemu error info
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+    # parse the usb toplogy from cfg
+    parse_usb_topology(params)
+    logging.info("starting vm according to the usb toplogy")
+    if params.get("multi_hubs_in_series_negative", "no") == "yes":
```

Comment:
Seems we do not need this check because the script is only introduced for negative tests.